editor/components/meta-boxes/meta-boxes-area/index.js

```diff
@@ -1,153 +1,85 @@
 /**
  * External dependencies
  */
-import { isEqual } from 'lodash';
 import classnames from 'classnames';
 import { connect } from 'react-redux';
-import jQuery from 'jquery';
 
 /**
  * WordPress dependencies
  */
-import { addQueryArgs } from '@wordpress/url';
 import { Component } from '@wordpress/element';
 import { Spinner } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
-import { handleMetaBoxReload, metaBoxStateChanged, metaBoxLoaded } from '../../../store/actions';
-import { getMetaBox, isSavingPost } from '../../../store/selectors';
+import { isSavingMetaBoxes } from '../../../store/selectors';
 
 class MetaBoxesArea extends Component {
+	/**
+	 * @inheritdoc
+	 */
 	constructor() {
 		super( ...arguments );
-
-		this.state = {
-			loading: false,
-		};
-		this.originalFormData = '';
 		this.bindNode = this.bindNode.bind( this );
-		this.checkState = this.checkState.bind( this );
-	}
-
-	bindNode( node ) {
-		this.node = node;
 	}
 
+	/**
+	 * @inheritdoc
+	 */
 	componentDidMount() {
-		this.mounted = true;
-		this.fetchMetaboxes();
+		this.form = document.querySelector( '.metabox-location-' + this.props.location );
+		this.node.appendChild( this.form );
```

Comment:
Not sure if we care to be guarded here, but if I pass an invalid `location` prop, this will error.